### PR TITLE
Design

### PR DESCRIPTION
**Pull request by Tymac15**

### CODE REVIEW

**Review by u20612894 — DISMISSED** [review]
Nothing out of place. Approved

**Review by ShashinGounden — DISMISSED** [review]
Looks good, what we agreed upon

**Comment by Bandisa** [comment]
All looks good. 
Thank you for the updated Admin page. Looks nice.

File: Design/Front end/UI designs/Web view/Edit module info page.jpg
Comment (Bandisa) [inline Design/Front end/UI designs/Web view/Edit module info page.jpg:1]: This page looks good.

File: Design/Front end/UI designs/readme.md
Comment (Bandisa) [inline Design/Front end/UI designs/readme.md:1]: Thank you for the update.

**Review by Bandisa — APPROVED** [review]
All looks good.
Thank you for the updated Admin page. Looks nice.

**Review by ShashinGounden — APPROVED** [review]

**Review by ShashinGounden — APPROVED** [review]

**Pull request merged**